tracer/src/Datadog.Trace/AppSec/RcmModels/RemoteConfigurationStatus.cs

```diff
@@ -13,11 +14,26 @@ namespace Datadog.Trace.AppSec.RcmModels;
 
 internal class RemoteConfigurationStatus
 {
-    internal List<RuleOverride> RulesOverrides { get; } = new();
+    internal Dictionary<string, List<RuleOverride>> RulesOverridesByFile { get; } = new();
 
-    internal List<RuleData> RulesData { get; } = new();
+    internal List<RuleOverride> RulesOverrides
+    {
+        get { return RulesOverridesByFile.SelectMany(x => x.Value).ToList(); }
```

Comment:
I don't think this is acceptable. Each time someone will use the property, they'll get a list allocated. A good example is the Debug log in Security.cs (we use twice the property in 4 lines). 
You should remove this property, and move the logic on the caller side. 
Same for the 2 other properties below

Reply:
I agree, I was trying to minimize the changes, but it would be better to remove the properties and move the work to the caller. Done in 995730f4b5c43d00fc92cf0d4231e2b7c761f62f